src/components/CreateProjectForm.vue

```diff
@@ -166,11 +167,20 @@
                     >
                     <b-input class="project-input" type="number" min="-360" max="360" v-model="pa"/>
                 </b-field>
+                <b-field 
+                    v-if="showAdvancedInputs"
+                    label="Meridian Flip">
+                    <b-checkbox v-model="noflip">No Flip</b-checkbox>
+                </b-field>
                 <b-field 
                     v-if="showAdvancedInputs"
                     label="Autofocus">
                     <b-checkbox v-model="frequent_autofocus">focus more frequently</b-checkbox>
                 </b-field>
+                <b-field 
```

Comment:
Is there a way to template out this a bit so that you don't have so many copy/paste sections? I know this is often how frontends work but it was a thought.

Reply:
Good suggestion. In this case, I suspect that reducing the line count comes at the cost of higher complexity. But I will keep that idea in mind.